src/Core/Plugin.php

```diff
@@ -186,10 +186,14 @@ protected static function _loadConfig()
         if (Configure::check('plugins')) {
             return;
         }
-        try {
-            Configure::load('plugins');
-        } catch (\Exception $e) {
+
+        $vendorFile = ROOT . DS . 'vendor' . DS . 'cakephp-plugins.php';
```

Comment:
This file will be in the vendors dir - so I don't think ther's a need for a config/constant - I can change it to use `dirname(dirname(dirname(__DIR__)))` instead.